kubeclient.gemspec

```
@@ -34,5 +34,5 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'rest-client', '~> 2.0'
   spec.add_dependency 'recursive-open-struct', '~> 1.0', '>= 1.0.4'
-  spec.add_dependency 'http', '~> 3.0'
+  spec.add_dependency 'http', '~> 4.0'
```

Can you change this to:
```suggestion
  spec.add_dependency 'http', '>= 3.0', '< 5.0'
```

Hopefully that'd automagically make ruby2.2 to use http 3 (http 4 declares `required_ruby_version = ">= 2.3"`).
And leave the option to use 3 for anyone that has other constraints.